tests/integration/service_test.py

```diff
@@ -273,6 +273,30 @@ def test_execute_convergence_plan_recreate(self):
                           self.client.inspect_container,
                           old_container.id)
 
+    def test_execute_convergence_plan_recreate_twice(self):
+        service = self.create_service(
+            'db',
+            volumes=[VolumeSpec.parse('/etc')],
+            entrypoint=['top'],
+            command=['-d', '1'])
+
+        orig_container = service.create_container()
+        service.start_container(orig_container)
+
+        orig_container.inspect()  # reload volume data
+        volume_path = orig_container.get_mount('/etc')['Source']
+
+        # Do this twice to reproduce the bug
+        for _ in xrange(2):
```

Comment:
Doesn't exist in python 3. Just use `range(2)` instead - we probably have enough RAM for those 12 bytes.